app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/SignDLCDialog.scala

```diff
@@ -1,190 +1,220 @@
+//                       text = str.outcome
+//                       editable = false
+//                     },
+//                     0,
+//                     nextRow)
+//        gridPane.add(new TextField() {
+//                       text = satoshis.toString
+//                       editable = false
+//                     },
+//                     1,
+//                     nextRow)
+//        nextRow += 1
+//      }
+//
+//      gridPane.add(new Label("Fee Rate"), 0, nextRow)
+//      gridPane.add(new TextField() {
+//                     text = status.feeRate.toString
+//                     editable = false
+//                   },
+//                   1,
+//                   nextRow)
+//      nextRow += 1
+//
+//      gridPane.add(new Label("Refund Date"), 0, nextRow)
+//      gridPane.add(
+//        new TextField() {
+//          text = GUIUtil.epochToDateString(status.timeouts.contractTimeout)
+//          editable = false
+//        },
+//        1,
+//        nextRow)
+//      nextRow += 1
+//    }
+//
+//    acceptTLVTF.onKeyTyped = _ => {
+//      if (!dlcDetailsShown) {
+//        Try(
+//          LnMessageFactory(DLCAcceptTLV).fromHex(
+//            acceptTLVTF.text.value)) match {
+//          case Failure(_) => ()
+//          case Success(lnMessage) =>
+//            val tempId = lnMessage.tlv.tempContractId
+//            dlcs.find(_.tempContractId == tempId) match {
+//              case Some(dlc) =>
+//                dlc.contractInfo.contractDescriptor.toTLV match {
+//                  case _: ContractDescriptorV0TLV =>
+//                    dlcDetailsShown = true
+//                    showDLCTerms(dlc)
+//                    dialog.dialogPane().getScene.getWindow.sizeToScene()
+//                  case _: ContractDescriptorV1TLV =>
+//                    () // todo not supported
+//                }
+//              case None => ()
+//            }
+//        }
+//      }
+//    }
+//
+//    dialog.dialogPane().content = new ScrollPane {
+//      content = new VBox(gridPane)
+//    }
+//
+//    // When the OK button is clicked, convert the result to a SignDLC.
+//    dialog.resultConverter = dialogButton =>
+//      if (dialogButton == ButtonType.OK) {
+//
+//        val acceptHex = acceptTLVTF.text.value
+//        val accept = LnMessageFactory(DLCAcceptTLV).fromHex(acceptHex)
+//
+//        Some(SignDLC(accept))
+//      } else None
+//
+//    val result = dialog.showAndWait()
+//
+//    result match {
+//      case Some(Some(cmd: SignDLCCliCommand)) =>
+//        Some(cmd)
+//      case Some(_) | None => None
+//    }
+//  }
+
+  override def constructFromInput(
+      inputs: Predef.Map[String, Node]): SignDLCCliCommand = {
+    DLCDialog.acceptDLCFile match {
+      case Some(file) =>
+        DLCDialog.acceptDLCFile = None // reset
+        DLCDialog.acceptFileChosenLabel.text = "" // reset
+        SignDLCFromFile(file.toPath, destination = None) //is none right here?
```

Comment:
destination is None if you aren't writing the result to a file, otherwise I believe it is supposed to contain the path